nsv13/code/modules/overmap/physics.dm

```diff
@@ -568,6 +568,8 @@ This proc is to be used when someone gets stuck in an overmap ship, gauss, WHATE
 	return ..()
 
 /obj/structure/overmap/proc/fire_projectile(proj_type, atom/target, homing = FALSE, speed=null, user_override=null, lateral=FALSE, ai_aim = FALSE, miss_chance=5, max_miss_distance=5) //Fire one shot. Used for big, hyper accelerated shots rather than PDCs
+	if(!z || QDELETED(src))
```

Comment:
```suggestion
	if(!loc || QDELETED(src))
```
wouldn't you just want to check for the null loc? z=0 when you're in the contents of an object

Reply:
z is 0 when you aren't on a turf and there is no case where a overmap should be not on a turf and firing, is there?

Reply:
> z is 0 when you aren't on a turf and there is no case where a overmap should be not on a turf and firing, is there?

There *shouldn't* but just to be safe y'know.

Reply:
... this *is* the "to be safe" option. There is no occasion any overmap should be *firing* while not on a turf.